app/controllers/api/migration_analytics_manifest_controller.rb

```diff
@@ -0,0 +1,34 @@
+def parse_json_if_valid(json)
+    return JSON.parse(json)
+  rescue JSON::ParserError => e
+    return :invalid
+end
+
+module Api
+  class MigrationAnalyticsManifestController < BaseController
+    def index
+      tmp_manifest_path = '/tmp/migration-analytics-manifest.json'
+      tmp_manifest = File.file?(tmp_manifest_path) ? parse_json_if_valid(File.read(tmp_manifest_path)) : nil
+      tmp_manifest_valid = tmp_manifest != nil && tmp_manifest != :invalid
+
+      if tmp_manifest_valid
+        manifest_path = tmp_manifest_path
+        manifest = tmp_manifest
+      else
+        manifest_path = File.join(
+          Bundler.environment.specs['cfme-migration_analytics'].first.full_gem_path,
```

Comment:
Think you can just do `Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")`

Reply:
Ah, that is better, thanks.

Also, I notice that examples from you and @Fryguy all use double-quotes and I've been using single-quotes in all my Ruby so far, just because that's what we did in v2v. Do you guys have an opinion on which should be used? I can switch them all over to double-quotes.

Reply:
That's mostly a personal preference, in ruby you can't do string interpolation in a string with single quotes which some people think is a good thing (i.e. it indicates the string is a constant).

Personally I think it looks sloppy having a mixture of `''` and `""`.  You have to use `""` if you do any string interpolation, so I tend to just use `""` everywhere.

Reply:
Sounds good. I agree it's sloppy to have both. Changed them all to `""`.

Reply:
I pushed this commit too late to be merged in this PR, so I added it to https://github.com/RedHatCloudForms/cfme-migration_analytics/pull/10.